datasets/cedr/cedr.py

```diff
@@ -0,0 +1,174 @@
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://github.com/sag111/CEDR"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = {
+    "main": "https://drive.google.com/u/0/uc?id=1flukWj075o7InAlBZm7htXEqZM6D-LfA&export=download",
+    "enriched": "https://drive.google.com/u/0/uc?id=1w9fJf9SwxtEABy5uhougn_2ZlxKBbZHV&export=download",
+}
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class cedr(datasets.GeneratorBasedBuilder):
+    """This dataset is designed to solve emotion recognition task for text data in Russian."""
+
+    VERSION = datasets.Version("0.1.1")
+
+    # This is an example of a dataset with multiple configurations.
+    # If you don't want/need to define several sub-sets in your dataset,
+    # just remove the BUILDER_CONFIG_CLASS and the BUILDER_CONFIGS attributes.
+
+    # If you need to make complex sub-parts in the datasets with configurable options
+    # You can create your own builder configuration class to store attribute, inheriting from datasets.BuilderConfig
+    # BUILDER_CONFIG_CLASS = MyBuilderConfig
+
+    # You will be able to load one or the other configurations in the following list with
+    # data = datasets.load_dataset('my_dataset', 'first_domain')
+    # data = datasets.load_dataset('my_dataset', 'second_domain')
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="main", version=VERSION, description="This part of CEDR dataset covers a main version"
+        ),
+        datasets.BuilderConfig(
+            name="enriched", version=VERSION, description="This part of CEDR dataset covers a enriched version"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "main"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        # TODO: This method specifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        if self.config.name == "main":  # This is the name of the configuration selected in BUILDER_CONFIGS above
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "labels": datasets.features.Sequence(datasets.Value("string")),
+                    "source": datasets.Value("string"),
+                    # These are the features of your dataset like images, labels ...
+                }
+            )
+        else:  # This is an example to show how to have different features for "first_domain" and "second_domain"
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "labels": datasets.features.Sequence(datasets.Value("string")),
```

Comment:
```suggestion
                    "labels": datasets.features.Sequence(datasets.ClassLabel(names=["joy," "sadness", "surprise", "fear", "anger"])),
```

after this change you will have to regenerate the dataset_infos.json file